app/src/test/java/org/simple/clinic/newentry/PatientEntryEffectHandlerTest.kt

```diff
@@ -100,6 +100,20 @@ class PatientEntryEffectHandlerTest {
     verifyZeroInteractions(validationActions)
   }
 
+  @Test
+  fun `when fetch colony or village names effect is received, then fetch them from the database`() {
```

Comment:
Maybe instead of saying `then fetch them from the database`, we can say `then fetch colony or village names` or `then load colony or village names` so that the test is not tied to the implementation details. Like if we were to replace the implementation of getting to the database with something different, then the test will lose context.